_strategies/modular-procurement.md

```diff
@@ -10,37 +10,37 @@ image-credit: https://www.flickr.com/photos/aaronharmon/
+Modular procurement makes it easier to manage software development by segmenting risk. This acquistion strategy permits isolated failure in one unit of many units, rather than letting it impact the entire project. Because each project is smaller, they are easier to comprehend and manage, making problems and risks smaller so you can recognize and resolve them more easily.
 
-In practice, most traditional contracts already have multiple companies responsible for different parts of the effort, but you may not see that complexity until something goes wrong. The majority of large scale government IT projects fail, so it’s nearly inevitable that something will go wrong under a traditional procurement process.
+As the government needs to more closely monitor the performance of smaller contracts they can help remedy problems to avoid cost overlays and drawn out schedules.
 
-The [Standish Group found](http://www.computerworld.com/article/2486426/healthcare-it/healthcare-gov-website--didn-t-have-a-chance-in-hell-.html) that:
+In contrast, the vast majority of larger projects are overbudget or failing. The [Standish Group found](http://www.computerworld.com/article/2486426/healthcare-it/healthcare-gov-website--didn-t-have-a-chance-in-hell-.html) that:
 
 > Of 3,555 projects from 2003 to 2012 that had labor costs of at least $10 million, only 6.4% were successful. The Standish data showed that 52% of the large projects were "challenged," meaning they were over budget, behind schedule or didn't meet user expectations. The remaining 41.4% were failures — they were either abandoned or started anew from scratch.
 
+### Shorter contracts will reduce the risk of factors beyond your control.
 
-#### Modular procurement doesn't necessarily mean more work for your procurement team.
+Planning work for a successful six-month to twelve-month contract is easier than planning a successful six-year contract. Modular contracting allows you to craft procurements that set out expectations for segments of work rather than scope out every contigency that could happen over the life of the product. Longer contracts also don't easily account for inevitable changes in leadership, legislative and other policy constraints or emerging technologies.
 
-You do need to do some work up-front to change your procurement processes to support modular procurement. Modular procurement can actually reduce the work required, such as by simplifying vendor selection and speeding up the process.
+To do this modular procurement should model iterative software development practices. This means that you are continuously improving both the product and your process and delivering some customer value as soon as possible. Building a Minimal Viable Product (MVP) and learning from it ultimately results in a better product than the traditional approach of trying to scope out what an complex product without validated assumptions looks like all at once.
```

Comment:
Suggest: "To do this, modular procurement should model iterative software development practices. This means that you will continuously improve both the product and your process and deliver customer value as soon as possible. Building a Minimal Viable Product (MVP) and iterating on it ultimately results in a better product than the traditional approach of scoping out a complex product all at once."